### ISSUE TITLE
Cop idea: useless `ruby2_keywords`

### ISSUE THREAD

**Issue by ojab** [rubocop/rubocop#10202]
If method doesn't have kwargs, `ruby2_keywords` is useless, but sometime is used due to overlook. Rubocop should warn about
```
def xxx(foo)
end
ruby2_keywords :xxx
```
and alike to reduce useless code amount.
